helix-core/src/main/java/org/apache/helix/cloud/event/HelixEventHandlingUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.helix.cloud.event;
+
+import org.apache.helix.BaseDataAccessor;
+
+
+public class HelixEventHandlingUtil {
+
+  public static boolean enableInstanceForCloudEvent(String clusterName, String instanceName, boolean isEnable, long timeStamp,
+      BaseDataAccessor dataAccessor) {
+    return true;
+  }
+
+  public static boolean IsInstanceDisabledForCloudEvent(String clusterName, String instanceName,
```

Comment:
I understand why you name it like that - so it's more clear that this check is "for cloud event". But it is a bit counter intuitive because we return true for disabled, and return false for enabled. I'm thinking maybe we could still name it "enableInstance", "isInstanceEnabled", etc. But stress that these methods are for use cases under the cloud event context only in the method java doc, and should not be used for other purposes. What do you think?

Reply:
Based on our offline discussion, let's keep the current name..